R/selectors.R

```diff
@@ -982,7 +982,7 @@ cs__starts_with <- function(...) {
 
 #' Select all String (and, optionally, Categorical) string columns.
 #'
-#' @inheritParams rlang::check_dots_empty0
+#' @inheritParams rlang::args_dots_empty
```

Comment:
Why this change? Personally I prefer the "Dots which should be empty." from `check_dots_empty0()` (straight to the point) over "These dots are for future extensions and must be empty." from `args_dots_empty()` (explains why they should be empty, which isn't necessary IMO).

If we end up going with your change, it should be applied everywhere throughout the docs.

Reply:
Because that is what `check_dots_empty` recommends.
https://rlang.r-lib.org/reference/check_dots_empty.html

> If we end up going with your change, it should be applied everywhere throughout the docs.

I already did that in 4d1b6993118f182e50ea28dd8556e114a2899ceb. (I had noticed this in the review phase of a past PR, but it wasn't a big deal so I just merged it in with the intention of fixing it myself later)

Reply:
Ok, thanks